__tests__/app.test.js

```diff
@@ -196,8 +197,52 @@ describe("GET /api/articles",()=>{
             })
         })
     })
+    describe("Articles topic query tests",()=>{
+        it("Should return the articles that have the topic (mitch)",()=>{
+            return request(app)
+            .get("/api/articles?topic=mitch")
+            .expect(200)
+            .then(({body})=>{
+                body.articles.forEach(article => {
```

Comment:
I would add a test for the length of the array before using a forEach. If the array is empty then the tests inside the forEach would be skipped but then length test outside would still run